component/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Fix the license year.
 Don't use apache license (use the GPL license). Check other places as well.

Reply:
Updated licenses in [#045ea86c0689a8343f00de3b61249053414144d0](https://github.com/wso2-extensions/siddhi-gpl-execution-streamingml/pull/16/commits/045ea86c0689a8343f00de3b61249053414144d0)